ql/termstructures/volatility/swaption/swaptionvolcube.cpp

```diff
@@ -76,8 +76,6 @@ namespace QuantLib {
                    swapIndexBase_->tenor() << ")");
 
         registerWithVolatilitySpread();
-        registerWith(Settings::instance().evaluationDate());
-        evaluationDate_ = Settings::instance().evaluationDate();
```

Comment:
This is because it's already done in the base class, right?  We still need a notification if the evaluation date changes, don't we?

Reply:
Yes I think it is done in the `TermStructure` constructor.